spring-cloud-deployer-local/src/main/java/org/springframework/cloud/deployer/spi/local/AbstractLocalDeployerSupport.java

```diff
@@ -54,6 +54,10 @@
 	private static final String USE_SPRING_APPLICATION_JSON_KEY =
 			LocalDeployerProperties.PREFIX + ".use-spring-application-json";
 
+	public static final String SERVER_PORT_KEY = "server.port";
+
+	public static final String SERVER_PORT_KEY_PREFIX = "--" + SERVER_PORT_KEY + "=";
```

Comment:
The "prefix" prepending `--` and adding `=` seems a bit odd.

Reply:
We could also keep them at package level vs. public

Reply:
I just moved the existing field, but I agree that the naming doesn't make sense...I can fix if you want.